xbmc/music/MusicDatabase.h

```diff
@@ -429,8 +425,9 @@ class CMusicDatabase : public CDatabase
   /////////////////////////////////////////////////
   // Scraper
   /////////////////////////////////////////////////
-  bool SetScraperForPath(const std::string& strPath, const ADDON::ScraperPtr& info);
-  bool GetScraperForPath(const std::string& strPath, ADDON::ScraperPtr& info, const ADDON::TYPE &type);
+  bool SetScraper(int id, const CONTENT_TYPE &content, const ADDON::ScraperPtr& scraper);
```

Comment:
Changed SetScraper and SetScraperAll, however left GetScraper using `ADDON::ScraperPtr& scraper` as otherwise it doesn't work. I am reusing historic code so perhaps that needs to change in some other way?